app/Repositories/Legacy/Two/SignupRepository.php

```diff
@@ -49,16 +49,16 @@ public function create($data)
      *
      * @param  string $northstarId
      * @param  int $campaignId
-     * @param  int $campaignRunId
      * @return \Rogue\Models\Signup|null
      */
-    public function get($northstarId, $campaignId, $campaignRunId)
+    public function get($northstarId, $campaignId)
     {
         $signup = Signup::where([
             'northstar_id' => $northstarId,
             'campaign_id' => $campaignId,
-            'campaign_run_id' => $campaignRunId,
-        ])->first();
+        ])->orderByRaw('campaign_run_id IS NULL')
```

Comment:
ah interesting! So `orderByRaw` just makes sure that the `NULL` values are at the bottom?

Reply:
Oh! It should be the exact opposite, that was a mistake. Updated in c7deaa4.

The idea is that if we make new signups where `campaign_run_id` is null, we want those to be prioritized over old ones where this is a number. So the final query looks like kinda like this:

```sql
SELECT * FROM signups
WHERE northstar_id = "5543dfd6469c64ec7d8b46b3" AND campaign_id = 1144
ORDER BY campaign_run_id IS NULL DESC, `campaign_run_id` DESC LIMIT 1
```

Reply:
@chloealee I added a new test for this in 9d794d2. Thanks for catching!